themes/generic/src/main/assets/js/warnOldVersion.js

```diff
@@ -0,0 +1,22 @@
+function initOldVersionWarnings($, thisVersion, projectUrl) {
+    var schemeLessUrl = projectUrl;
```

Comment:
To avoid double slash at the end, you can make sure that `schemeLessUrl` ends with a slash by adding `.replace(/\/?$/, '/')`.

This is from https://stackoverflow.com/questions/11531363/javascript-jquery-add-trailing-slash-to-url-if-not-present